packages/datatrak-web-server/src/types.ts

```diff
@@ -2,17 +2,25 @@
  * Tupaia
  * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
  */
-import { ServerBoilerplateModelRegistry } from '@tupaia/server-boilerplate';
+import { ModelRegistry } from '@tupaia/database';
+import {
+  CountryModel,
+  EntityModel,
+  FeedItemModel,
+  OneTimeLoginModel,
+  OptionModel,
+  SurveyModel,
+  SurveyResponseModel,
+  UserModel,
+} from '@tupaia/server-boilerplate';
 
-type Models =
-  | 'database'
-  | 'user'
-  | 'entity'
-  | 'country'
-  | 'feedItem'
-  | 'survey'
-  | 'surveyResponse'
-  | 'oneTimeLogin'
-  | 'option';
-
-export type DatatrakWebServerModelRegistry = Pick<ServerBoilerplateModelRegistry, Models>;
+export interface DatatrakWebServerModelRegistry extends ModelRegistry {
+  readonly user: UserModel;
+  readonly entity: EntityModel;
+  readonly country: CountryModel;
+  readonly feedItem: FeedItemModel;
+  readonly survey: SurveyModel;
+  readonly surveyResponse: SurveyResponseModel;
+  readonly oneTimeLogin: OneTimeLoginModel;
+  readonly option: OptionModel;
+}
```

Comment:
There's a bit more boilerplate, but I think I prefer this interface from a readability standpoint